### DISCUSSION TITLE
Strimzi kafka Connect Load Testing and benchmarking

### DISCUSSION THREAD

**Discussion by Abdul1929** [strimzi/strimzi-kafka-operator#5973] [Q&A]
Hi Team,

I am using strimzi operator 0.26.0 and have kafka and kafka connect deployment version 2.8.0. Do we have any docs or list of steps to perform load testing and benchmarking for kafka connect.

I am checking this in my test environment where my source DB is mongodb and I want to put load from mongodb to kafka and checked with different test case like 2cpu 4gb ram, 4cpu 4gb ram and check the load from multiple connectors also and perform benchmarking.

**Comment by scholzj** [comment]
There is nothing what would be part of the Strimzi docs. Keep in mind that every connector is different and there are hundreds of connectors - so it will probably differ for each connector. But Connect is still Connect => so I think if you look for any generic guide for Kafka Connect, you should be able to use it with Connect deployed with Strimzi.

**Reply by Abdul1929** [reply]
okay thanks.. I have test case for apache kafka connect which I can take reference for strimzi.
